tools/cldr-apps/js/src/views/TransferVotes.vue

```diff
@@ -0,0 +1,369 @@
+        message="Source and Target locales differ"
+        show-icon
+      />
+
+      <a-input
+        v-if="fromLocale"
+        v-model:value="toLocale"
+        placeholder="Enter a target locale"
+        style="width: 20em"
+      >
+        <template #addonBefore> Target Locale: </template>
+      </a-input>
+      <p />
+      <b>
+        <button
+          v-on:click="transfer"
+          v-if="
+            fromLocale &&
+            toLocale &&
+            !started &&
+            votesCache[transferFromId] &&
+            !votesCache[transferFromId].loading
+          "
+        >
+          Copy to “{{ toLocale }}”
+        </button>
+      </b>
+
+      <p />
+
+      <a-alert
+        v-if="started"
+        type="info"
+        message="Copy Status"
+        v-model:description="started"
+        show-icon
+        @click="
+          started = null;
+          current = 3;
+        "
+      />
+    </div>
+    <hr />
+  </div>
+</template>
+
+<script>
+import { notification } from "ant-design-vue";
+import * as cldrText from "../esm/cldrText";
+import { ref, reactive } from "vue";
+import * as cldrLoad from "../esm/cldrLoad";
+
+function errBox(message) {
+  console.error("TransferVotes.vue: " + message);
+  notification.error({
+    description: "Problem with Copy Votes",
+    message,
+    placement: "topLeft",
+    duration: 0,
+  });
+}
+
+export default {
+  setup() {
+    const qparams = new URLSearchParams(document.location.search);
+    const transferToId = qparams.get("transferTo");
+    const transferFromId = qparams.get("transferFrom") || 0; // for URL based updating
+    let current = 1;
+    return {
+      current: ref(current),
+      transferToId: ref(transferToId),
+      chooseUserList: reactive({}),
+      fullUserList: reactive({}),
+      votesCache: reactive({}),
+      transferFromVotes: ref(null),
+      transferToInfo: ref(null),
+      transferFromId: ref(transferFromId),
+      started: ref(null),
+      fromLocale: ref(null),
+      toLocale: ref(null),
+    };
+  },
+  created() {
+    this.fetchData();
+  },
+  methods: {
+    cancel() {
+      this.started = null;
+      this.transferFromId = 0;
+      this.current = 1;
+      this.fromLocale = null;
+      this.toLocale = null;
+    },
+    goBack() {
+      // go back to list users page
+      this.current = 0;
+      window.location.replace("v#list_users");
+    },
+    setFromVotes(id) {
+      this.transferFromVotes = this.votesCache[id];
+      if (!this.transferFromVotes) {
+        this.fetchFromVotes(id);
+      }
+    },
+    fetchFromVotes(id) {
+      // Don't warn if the id was already different from the user choice
+      const quiet = id != this.transferFromId;
+      if (!this.votesCache[id]) {
+        this.votesCache[id] = { loading: true };
+        const sessionId = this.$cldrOpts.cldrStatus.getSessionId();
+        fetch(`SurveyAjax?what=user_oldvotes&s=${sessionId}&old_user_id=${id}`)
+          .then((r) => r.json())
```

Comment:
suggestions for later improvement: use cldrAjax.doFetch, and similarly encapsulate the SurveyAjax dependency in cldrAjax; use URLSearchParams for constructing url; import cldrStatus like any other import, don't give handle it exceptionally with $cldrOpts


maybe more work and more code in the short run, worth it since more maintainable in the long run